src/projectscene/qml/Audacity/ProjectScene/trackspanel/TrackItem.qml

```diff
@@ -171,6 +171,14 @@ ListItemBlank {
 
                 spacing: 16
 
+                opacity: root.height  > root.mapFromItem(this, 0, height + bottomSeparator.height).y ? 1 : 0
```

Comment:
looks good but I expected it to be visible property

Reply:
Can we animate on `visible`? If I set both `visible` and `opacity` to 0, it results in flickering.